CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# dbt_stripe_source v0.15.0
+
+[PR #95](https://github.com/fivetran/dbt_stripe_source/pull/95) includes the following updates:
+
+## Breaking Change for dbt Core < 1.9.5
+> *Note: This is not relevant to Fivetran Quickstart users.*
+Migrated `freshness` from a top-level source property to a source `config` in alignment with [recent updates](https://github.com/dbt-labs/dbt-core/issues/11506) from dbt Core. This will resolve the following deprecation warning that users running dbt >= 1.9.5 may have received:
```

Comment:
```suggestion
Migrated `freshness` from a top-level source property to a source `config` in alignment with [recent updates](https://github.com/dbt-labs/dbt-core/issues/11506) from dbt Core. This will resolve the following deprecation warning that users running dbt >= 1.9.6 may have received:
```